charts/nitro/templates/tests/test.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: "{{ include "nitro.fullname" . }}-ci-configmap"
+data:
+  config.yaml: |
+    {{- toYaml .Values.ci | nindent 4 }}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: "ci-secret"
+type: Opaque
+data:
+  {{- range $key, $value := .Values.ci.secrets }}
+  {{ $key }}: {{ $value | b64enc | quote }}
+  {{- end }}
+---
+
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ include "nitro.fullname" . }}-test-connection"
+  labels:
+    {{- include "nitro.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": test
+spec:
+  containers:
+  - name: sre-nitro-test-suite
+    image: public.ecr.aws/f6s7v9z5/offchain-labs/sre-nitro-test:v0.0.6
+    {{- if .Values.ci.debug.enabled }}
+    command: ["/bin/sh", "-c", "--"]
+    args: ["while true; do sleep 30; done;"]
```

Comment:
shouldn't we be running the test tool from the image rather than sleeping?

Reply:
Thats only run if `.Values.ci.debug.enabled ` is true. Making sure the secrets were loaded correctly required using this feature extensively 🙃

Reply:
ah whoops, sorry I missed that part, thanks for explaining :)